backend/Controllers/UserController.cs

```diff
@@ -28,19 +28,44 @@ public ActionResult<User> Register([FromBody] User user)
 
             using (var context = new Database())
             {
-                // Check if user already exists
                 var existingUser = context.Users.FirstOrDefault(u => u.id == user.id);
                 if (existingUser != null)
                 {
                     return Conflict("User already registered in the database.");
                 }
 
-                // Add user to database
                 context.Users.Add(user);
                 context.SaveChanges();
             }
 
             return CreatedAtRoute("user", new { id = user.id }, user);
         }
+
+        [HttpPost("login")]
+        public ActionResult<User> Login([FromBody] User user)
+        {
+            if (user.password == null || user.username == null || user == null)
```

Comment:
Żeby przyspieszać sprawdzanie warunków, można zastosować sprawdzanie od największego elementu. Tj. Najpierw czy istnieje user, jeśli istnieje to dopiero sprawdzamy hasło lub nazwę użytkownika

Reply:
podajże w C# mamy taką opcję sprawdzania typu "user.password is not null" Tak jak się pisze, z pominięciem tych podwójnych znaków równości

Reply:
o to spoczko poprawie jutro